misk/src/main/kotlin/misk/concurrent/RealExecutorServiceFactory.kt

```diff
@@ -62,31 +66,51 @@ internal class RealExecutorServiceFactory @Inject constructor(
     awaitAllTerminated.start()
   }
 
+  // The traceWithActiveSpanOnly=false flag below means we will create a new root span if there is
+  // not an ongoing active span at the point of handing over a task to the ExecutorService. This
+  // means we run the risk of having too many root spans but I prefer to err on the side of too much
+  // visibility. This may cause unnecessary noise though so we may need to change this as we try it
+  // out in production.
```

Comment:
I think this is a comment that wants to be a runtime parameter and some documentation?

Reply:
Yeah possibly. I'd like to merge this as is and then see how things look and possibly add a parameter if necessary. I hope that this setting is good for all our services but I may be wrong.